examples/Demo.Console.NET6/Program.cs

```diff
@@ -1,22 +1,24 @@
 // Copyright (c) Microsoft Corporation.

```

Comment:
We might need to adjust the configuration of stylecop here. This is the correct comment and we don't need XML surrounding it, but GitHub Actions is still complaining that
```
The file header XML is invalid.
```

Reply:
> StyleCop was installed on all projects except `Demo.Console.NET6` project. Hence, there were no stylecop errors. I installed it directly on the `Demo.Console.NET6` project, and hence there is an additional `stylecop.json` file. I removed that file now and I installed `StyleCop` on this project at the solution level which originally included `StyleCop` on all but this project.

To be clear, `StyleCop` at solution level looked like this. Changed it now.  
![image](https://user-images.githubusercontent.com/99682944/163644673-b662a67c-3cc8-4414-9851-cd7209005196.png)